dir.traversal.targets

```diff
@@ -33,6 +33,21 @@
     <Error Condition="'$(MSBuildLastTaskResult)'=='false'" />
   </Target>
 
+  <Target Name="GetFilesToPackage" DependsOnTargets="FilterProjects"
+          Returns="@(FilesToPackage)">
```

Comment:
Just copy pasting this common infrastructure from our other branches.  This gets used in builds files so those never show up in VS.